Resources/Prototypes/Entities/Objects/Specific/Janitorial/trashbag.yml

```diff
@@ -39,31 +39,65 @@
     size: Normal
 
 - type: entity
-  name: trash bag
+  parent: TrashBag
   id: TrashBagBlue
+  name: trash bag
```

Comment:
we should migrate this out if we're adding trash bag of holding. the sprite being the same is really wack